docs/ref/settings.txt

```diff
@@ -3108,8 +3108,8 @@ Whether to use ``HttpOnly`` flag on the session cookie. If this is set to
 cookie.
 
 HttpOnly_ is a flag included in a Set-Cookie HTTP response header. It's part of
-the :rfc:`6265` standard for cookies and can be a useful way to mitigate the
-risk of a client-side script accessing the protected cookie data.
+the :rfc:`6265#section-5.3` standard for cookies and can be a useful way to
```

Comment:
I think this should point to the `section-4.1.2.6` (like in the ` docs/ref/request-response.txt`) or in both cases we should leave `` :rfc:`6265` ``.

Reply:
You're right. `section-5.3` was a bad copy/paste I think.

I've updated the PR to point to section 4.1.2.6 as suggested.